index.bs

```diff
@@ -12032,22 +12034,18 @@ defined in section [[#platformobjectset]].
 </div>
 
 
-<h4 id="getownproperty">Platform object \[[GetOwnProperty]] method</h4>
+<h4 id="legacy-platform-object-getownproperty" oldis="getownproperty">Legacy platform object \[[GetOwnProperty]] method</h4>
```

Comment:
`oldids` let you keep around previous ids to avoid breaking x-refs (see [relevant Bikeshed doc](https://tabatkins.github.io/bikeshed/#id-gen)]). Agree that this is still somewhat descriptive, but I feel like it gives it way to much weight that way. This is the getOwnProp method for a relatively small subset of things.